source/Calamari.Tests/Fixtures/Certificates/ImportCertificateCommandFixture.cs

```diff
@@ -74,13 +78,14 @@ public void NoStoreLocationProvided_StoresInUserName()
             // Hygiene Cleanup
             cert.EnsureCertificateNotInStore(storeName, storeLocation);
         }
-        
+#endif        
+
         CalamariResult Invoke(VariableDictionary variables)
         {
             using (var variablesFile = new TemporaryFile(Path.GetTempFileName()))
             {
                 variables.Save(variablesFile.FilePath);
-                return Invoke(Calamari()
+                return InvokeInProcess(Calamari()
```

Comment:
This simply runs the test in the same process as the test harness, making debugging possible. 
I'd like to move all Calamari tests to this format now that we have a bit more of a handle on the different runtimes that we are actually running